modules/installation-aws-user-infra-requirements.adoc

```diff
@@ -156,6 +156,19 @@ If `m4` instance types are not available in your region, such as with
 
 You might be able to use other instance types that meet the specifications of these instance types.
 
+[id="csr-management-aws_{context}"]
+== Certificate signing requests management
+
+Because your cluster has limited access to automatic machine management when you
+use infrastructure that you provision, you must provide a mechanism for approving
+cluster certificate signing requests (CSRs) after installation. The
+`kube-controller-manager` only approves the kubelet client CSRs. The
+`machine-approver` cannot guarantee the validity of a serving certificate
+that is requested by using kubelet credentials because it cannot confirm that
+the correct machine issued the request. You must determine and implement a
+method of verifying the validity of the kubelet serving certificate requests
```

Comment:
You must determine and implement a method of verifying the Node's validity, and either approve or deny the certificate signing request(s) requested by the infra node(s).